src/hotspot/share/runtime/arguments.cpp

```diff
@@ -3948,7 +3948,7 @@ jint Arguments::parse(const JavaVMInitArgs* initial_cmd_args) {
   const NMT_TrackingLevel lvl = NMTUtil::parse_tracking_level(NativeMemoryTracking);
   if (lvl == NMT_unknown) {
     jio_fprintf(defaultStream::error_stream(),
-                "Syntax error, expecting -XX:NativeMemoryTracking=[off|summary|detail]", nullptr);
+                "Syntax error, expecting -XX:NativeMemoryTracking=[off|summary|detail]");
```

Comment:
I think it also needs a trailing `\n`

Reply:
Good point. I triggered this error and it didn't look pretty:
```
> java.exe -XX:NativeMemoryTracking=blah
Syntax error, expecting -XX:NativeMemoryTracking=[off|summary|detail]Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```
I noticed a few other spots in this file where trailing `\n` is missing. I'll fix them here as well.